harbors/management/commands/create_harbors_from_json.py

```diff
@@ -0,0 +1,52 @@
+"""
+This command creates or updates Harbor objects based on a JSON
+that has the following structure for its items:
+
+"AIRORANTA": {
+    "servicemap_id": "40393",
+    "berth_count": 11,
+    "max_length": 500,
+    "max_width": 200
+}, ...
+
+"""
+import json
+
+from django.core.management.base import BaseCommand
+from django.utils.text import slugify
+
+from harbors.models import Harbor
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('--file', action='store', dest='file',
+                            help="Path to JSON file with harbors' data")
+
+    def handle(self, **options):
+        json_filepath = options['file']
+        if not json_filepath:
+            self.stderr.write('No path to JSON file provided')
```

Comment:
Probably better to raise [CommandError](https://docs.djangoproject.com/en/2.1/howto/custom-management-commands/#django.core.management.CommandError) here?